client/src/components/Projects/ProjectsPage.js

```diff
@@ -138,6 +138,10 @@ const ProjectsPage = ({ account, contentContainerRef }) => {
     setCopyModalOpen(true);
   };
 
+  const updateProjects = async () => {
```

Comment:
This seems sort of redundant with the useProjects hook, since both are calling the same projectService.get method. Maybe we should consider modifying the useProjects hook to support additional features like re-executing the query.  We have upcoming issues to filter projects by various criteria (hidden/visible, draft, snapshot, active/deleted, possibly by owner, etc.) which can probably be done front-end. Maybe the filtering is included in the hook?